### PR TITLE
Move frequency domain analysis to workbench

### LINKED ISSUE #25576: Move Frequency Domain Analysis GUI to workbench
The Frequency Domain analysis GUI should be ported to workbench.

### PR DESCRIPTION
**Pull request by Matthew-Andrew**
Merge after #25573

**Description of work.**

This moves the Frequency Domain Analysis Tab to the workbench.

**To test:**

In the workbench open the FDA GUI. It is in the Muon subsection of the interface list. Check that a few standard workflows work. If you need an explanation of how the GUI is supposed to work feel free to message me.

Fixes #25576

---

#### Reviewer ####

Please comment on the following ([full description](http://developer.mantidproject.org/IndividualTicketTesting/)):

##### Code Review #####

- Is the code of an acceptable quality?
- Does the code conform to the [coding standards](http://developer.mantidproject.org/Standards/)?
- Are the unit tests small and test the class in isolation?
- If there are changes in the release notes then do they describe the changes appropriately?

##### Functional Tests #####

- Do changes function as described? Add comments below that describe the tests performed?
- Do the changes handle unexpected situations, e.g. bad input?
- Has the relevant (user and developer) documentation been added/updated?

Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.

### CODE REVIEW

**Comment by AnthonyLim23** [comment]
had a play and it seems to work. @Mantid-Matthew you have conflicts

**Comment by AnthonyLim23** [comment]
need to test:

1. Works in mantidplot
2. what happens if you open GUI, let it drop to the back and then 'reopen' the GUI

**Comment by AnthonyLim23** [comment]
it appears that browse does not work in workbench. I browse to a file and select it. Then it does not load

**Comment by AnthonyLim23** [comment]
conflicts and phase table presenter is failing on OSX

**Review by AnthonyLim23 — DISMISSED** [review]
all works in workbench and mantidplot :shipit:

File: scripts/Muon/GUI/Common/grouping_tab_widget/grouping_tab_widget_view.py [hunk 1]
```diff
@@ -119,19 +120,24 @@ def get_description_text(self):
...
+        return show_file_browser_and_return_selection(self, file_filter, search_directories)[0]
 
     def show_file_save_browser_and_return_selection(self):
-        chosen_file = str(
-            QtWidgets.QFileDialog.getSaveFileName(
-                self,
-                "Select file",
-                '',
-                'XML files (*.xml)'))
+        if PYQT4:
```
Comment (martyngigg) [inline scripts/Muon/GUI/Common/grouping_tab_widget/grouping_tab_widget_view.py]: This check order looks good :+1:

File: scripts/Muon/GUI/Common/load_run_widget/load_run_view.py [hunk 2]
```diff
@@ -74,7 +74,7 @@ def setup_interface_layout(self):
         self.horizontal_layout.addWidget(self.increment_run_button)
 
         self.horizontal_layout.setContentsMargins(0, 0, 0, 0)
-        self.horizontal_layout.setMargin(0)
+        # self.horizontal_layout.setMargin(0)
```
Comment (martyngigg) [inline scripts/Muon/GUI/Common/load_run_widget/load_run_view.py]: Can we remove the commented out code?

File: scripts/Muon/GUI/MuonAnalysis/load_widget/load_widget_view.py [hunk 3]
```diff
@@ -21,39 +21,39 @@ def __init__(self, parent=None, load_run_view=None, load_file_view=None):
...
         self.horizontal_layout.setObjectName("horizontalLayout")
         self.horizontal_layout.addWidget(self.clear_button)
         self.horizontal_layout.addStretch(0)
         self.horizontal_layout.addWidget(self.multiple_loading_label)
         self.horizontal_layout.addWidget(self.multiple_loading_check)
 
         self.horizontal_layout.setContentsMargins(0, 0, 0, 0)
-        self.horizontal_layout.setMargin(0)
-        self.tool_widget = QtGui.QWidget(self)
+        # self.horizontal_layout.setMargin(0)
```
Comment (martyngigg) [inline scripts/Muon/GUI/MuonAnalysis/load_widget/load_widget_view.py]: Commented out code?

**Review by martyngigg — CHANGES_REQUESTED** [review]
Just a request for one small change to maintain future compatability.

**Review by martyngigg — APPROVED** [review]

**Pull request merged**